requirements.txt

```diff
@@ -1,3 +1,3 @@
 python-dateutil==2.4.2
 mock
--e git://github.com/openhealthcare/opal.git@v0.8.3#egg=opal
+-e git://github.com/openhealthcare/opal.git@v0.9.0#egg=opal
```

Comment:
If this targets Opal 0.9 then what version of Pathway is compatible with Opal 0.8.x ?